lib/src/tests.rs

```diff
@@ -358,3 +359,19 @@ fn test_stored_chunks_iter() {
     chunks_from_iter = list_stored_chunks(&repo).unwrap();
     assert_eq!(chunks_from_indexes.difference(&chunks_from_iter).count(), 0);
 }
+
+#[test]
+fn test_custom_chunking_size() {
+    let dir_path = rand_tmp_dir();
+    let chunking = RepoChunking {
```

Comment:
Maybe this could be in a loop, or at least a list of minium value, average and maximum?